minitests/roi_harness/create_design_json.py

```diff
@@ -46,6 +46,15 @@ def main():
     design_json['info'] = {}
     with open(args.design_txt) as f:
         for d in csv.DictReader(f, delimiter=' '):
+            if d['name'].startswith('dout['):
+                d['type'] = 'in'
```

Comment:
Ins't this backwards for the file format?  dout[x] is an output, not an input right?

Reply:
Ah yes, you're right.  Fixing now.